NetCoreRestApi/BusinessLayer/Manager/IProductManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿using BusinessLayer.Models;
+using System.Collections.Generic;
+
+namespace BusinessLayer.Manager
+{
+    interface IProductManager
+    {
+        public IEnumerable<ProductModel> GetAll();
```

Comment:
Remove public all methods are public in interface

Reply:
Done